src/script/view_model/content/AppLockViewModel.ts

```diff
@@ -46,28 +47,41 @@ export enum APPLOCK_STATE {
 
 const APP_LOCK_STORAGE = 'app_lock';
 
+const getInactivityTimeout = () => {
+  const teamState = container.resolve(TeamState);
+  const appLock = teamState.teamFeatures()['app-lock'];
+  if (appLock.status === 'enabled') {
+    return appLock.config.inactivity_timeout_secs;
+  }
+  return null;
+};
```

Comment:
Maybe this should be exported somewhere else to support more features..

Reply:
There can be a selector (`observables/computeds`) in the `teamState` (e.g. `isAppLockEnabled()` and `getAppLockInactivityTimeout()`)